docs/reference/cat/recovery.asciidoc

```diff
@@ -31,20 +34,30 @@ Now let's see what a live recovery looks like. By increasing the replica count
 of our index and bringing another node online to host the replicas, we can see
 what a live shard recovery looks like.
 
-[source,sh]
+[source,js]
+----------------------------------------------------------------------------
+PUT twitter/_settings 
+{
+  "number_of_replicas" : 2
+}
+
+GET _cat/recovery?v&h=i,s,t,ty,st,shost,thost,f,fp,b,bp
+----------------------------------------------------------------------------
+// CONSOLE
+// TEST[continued]
+
+[source,text]
 ----------------------------------------------------------------------------
-> curl -XPUT 'localhost:9200/wiki/_settings' -d'{"number_of_replicas":1}'
-{"acknowledged":true}
-
-> curl -XGET 'localhost:9200/_cat/recovery?v&h=i,s,t,ty,st,shost,thost,f,fp,b,bp'
-i     s t      ty      st    shost  thost  f     fp      b        bp
-wiki  0 1252ms store   done  hostA  hostA  4     100.0%  23638870 100.0%
-wiki  0 1672ms replica index hostA  hostB  4     75.0%   23638870 48.8%
-wiki  1 1698ms replica index hostA  hostB  4     75.0%   23348540 49.4%
-wiki  1 4812ms store   done  hostA  hostA  33    100.0%  24501912 100.0%
-wiki  2 1689ms replica index hostA  hostB  4     75.0%   28681851 40.2%
-wiki  2 5317ms store   done  hostA  hostA  36    100.0%  30267222 100.0%
+i       s t      ty             st   shost thost f fp     b        bp
+twitter 0 1252ms existing_store done n/a   hostA 4 100.0% 23638870 0.0%
```

Comment:
I'd prefer this tell a story in line with the first one but it doesn't really. Either we should use fake data and replace the whole time or we should use real data.